lib/internal/Magento/Framework/App/CacheInterface.php

```diff
@@ -22,10 +23,10 @@ interface CacheInterface
     public function getFrontend();
 
     /**
-     * Load data from cache by id
+     * Load Data from Cache by ID
      *
-     * @param  string $identifier
-     * @return string
+     * @param string $identifier
+     * @return bool|string
      */
     public function load($identifier);
 
```

Comment:
can you add php declare strict_types and declare return implicit at the end of method load function declare
Such as load(): ?string; for example